README.md

```diff
@@ -71,4 +75,24 @@ Authorization: 	Bearer <string>
 ```http request
 HTTP/1.1 403 Forbidden 
 HTTP/1.1 500 Internal server error
-``` 
\ No newline at end of file
+``` 
+
+## Features 📘 (incoming)
+
+- Start a go project
```

Comment:
I don't think the list is good.
The features of the app will be:
- As a user i want to read one articles
- As a admin i want to create an article
...